nautobot/docs/plugins/development.md

```diff
@@ -428,6 +428,53 @@ class AnimalType(DjangoObjectType):
 graphql_types = [AnimalType]
 ```
 
+#### Using GraphQL ORM Utility
```

Comment:
I think this should be a level-3 heading (`###`) not a level-four heading - it's a subheading of `## Integrating with GraphQL`, not a subheading of `### Creating Your Own GraphQL Type Object`.

```suggestion
### Using GraphQL ORM Utility
```